test/functional/tests/lazy_writes/cleaning_policy/test_cleaning_params.py

```diff
@@ -0,0 +1,199 @@
+#
+# Copyright(c) 2019-2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import time
+
+import pytest
+
+from api.cas import casadm
+from api.cas.cache_config import (
+    CacheMode,
+    CleaningPolicy,
+    FlushParametersAcp,
+    SeqCutOffPolicy,
+    FlushParametersAlru
+)
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_tools.dd import Dd
+from test_utils.os_utils import Udev
+from test_utils.size import Size, Unit
+from test_utils.time import Time
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_cleaning_policy_config():
```

Comment:
I think you should split this tests to three separate tests - that way we'll know what failed exactly at first glance

Reply:
This is such a basic test and should be so easy to analyze that I think it shouldn't require going through test prepare per each cleaning policy.